BTModules/Office365.psm1

```diff
@@ -0,0 +1,244 @@
+# Set strict mode.
+Set-StrictMode -Version 2.0
+
+# Disable progress bar.
+$ProgressPreference = "SilentlyContinue"
+
+# Set default ErrorAction to Stop.
+$ErrorActionPreference = "Stop"
+
+################################################################################
+# Clear all opened PSSessions.
+################################################################################
+function Clear-PSSession
+{
+    try
+    {
+        DisplayInProgressOperation "Remove PSSessions"
+        Get-PSSession | Remove-PSSession
+        DisplayCompletedOperation "OK" Green
+    }
+    catch
+    {
+        DisplayCompletedOperation "FAIL" Red
+        Write-Host -ForegroundColor Red "ERROR: Clear-PSSession failed with $_"
+        return $false
+    }
+}
+
+################################################################################
+# Connect to Office 365
+################################################################################
+function Connect-Office365($credentials)
+{
+    # Connect to MsolService
+    try
+    {
+        DisplayInProgressOperation "Connect to MsolService"
+        Connect-MsolService -Credential $credentials
+        DisplayCompletedOperation "OK" Green
+    }
+    catch
+    {
+        DisplayCompletedOperation "FAIL" Red
+        Write-Host -ForegroundColor Red "ERROR: Connect-MsolService failed with $_"
+        return $false
+    }
+
+    # Create new PSSession
+    try
+    {
+        DisplayInProgressOperation "Create New-PSSession"
```

Comment:
Because all other DisplayInProgressOperation outputs do not use dashes, maybe have this display "Create New PSSession" or "Create New PS Session".